### PR TITLE
To Do List CRUD

### PR DESCRIPTION
**Pull request by kennankole**

### CODE REVIEW

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## STATUS: CHANGES REQUIRED ♻️ 

## Hi @kennankole👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- GitHub flow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/index.js [hunk 1]
```diff
@@ -1,70 +1,100 @@
...
+				data.splice(index, listElem.length)
+				localStorage.setItem('listItems', JSON.stringify(data));
+			} else {
+				return;
+			}
+		})
+	} else {
+		return;
+	}
+}
```
[the remark this answers is not in the document]
Comment (kennankole) [reply src/index.js:91]: Hi, @thecodechaser , it is possible to delete one or multiple tasks once you highlight it as can be seen in the screencast below.

[Screencast from 06-10-2022  4:33:42 WB.webm](https://user-images.githubusercontent.com/39535023/194327231-6750b0ab-bdf8-455c-bbe9-32674016e7e8.webm)

File: src/index.js [hunk 2]
```diff
@@ -1,70 +1,100 @@
...
+	showData()
+} else {
+	localStorage.setItem('listItems', JSON.stringify(data));
+}
+
+// Check and uncheck items.
+const listItem = document.getElementsByClassName('list-text');
+const checked = document.getElementsByClassName('check');
+const elem = Array.from(checked);
+elem.forEach((item, index) => {
```
[the remark this answers is not in the document]
Comment (kennankole) [reply src/index.js:62]: Ok, I'll do that

**Review by Mihndim2020 — APPROVED** [review]
# STATUS: APPROVED!!!  🎉 🎉 🎉 :champagne: :champagne: :champagne:
Hi @kennankole, 

### Good job !!!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file

Your project is complete! Great work on making the changes requested by the previous reviewer. There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 🎉 🎉 :champagne: :champagne: :champagne:

Congratulations-Memes-FEATURED

## Optional suggestions

_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**